scripts/get_benchmark_results.py

```diff
@@ -1,17 +1,28 @@
 import os
 import sys
 
+from datetime import datetime, timedelta
+
+from scripts.get_summary import parse_environment_variables
 from scripts.google_cloud_storage import GoogleCloudStorage
 
 if __name__ == '__main__':
     output_bucket = os.getenv("OUTPUT_BUCKET")
 
+    parsed_variables = parse_environment_variables()
```

Comment:
I think it is fine to default output dir to `outputs` as part of `parse_environment_variables` then pass it to `get_files()`.